concatenate_flows/concatenate_pdms.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import csv
+import time
+import re
+from core_data_modules.traced_data import TracedData, Metadata
+from core_data_modules.traced_data.io import TracedDataJsonIO
+
+
+def key_normalisation(td_pdm, string_to_replace, replacement_string):
```

Comment:
Can you give this function more of an 'action' name, and be more descriptive on the normalisation e.g. `normalise_pdm_flow_keys`